versioned_docs/version-8.6/self-managed/concepts/elasticsearch-without-cluster-privileges.md

```diff
@@ -0,0 +1,282 @@
+---
+id: elasticsearch-without-cluster-privileges
+title: "Elasticsearch without cluster privileges"
+---
+
+In case the Camunda single application cannot access Elasticsearch with cluster privileges, it is possible to run the schema manager as a stand-alone application separate from the main application. The schema manager solely creates all necessary Elasticsearch Indices. In this case, elevated privileges are only required for the schema creation, the single application does not need to have cluster privileges to work any more.
+
+:::note Database Support
+This feature is only available from version 8.6.10 on and is also only supported for Elasticsearch installations (no OpenSearch support).
+:::
+
+:::note Essential privileges required by the single application
+An index-level privilege of at least `manage` is still required for the Camunda single application to work properly.
+:::
+
+## Setup
+
+For this setup to work, the database schema must be initialized first (step 1). This requires cluster level privileges for the database. This only needs to be executed once.
```

Comment:
:question: 

> This only needs to be executed once.

Wouldn't this be, this needs to be executed once on the first installation, and then every time they perform an update? Or are we not discussing updates?

Because they _will_ eventually update, and they _will_ need to run it again at that point.

Reply:
Do we have a way to ensure it will be done before the next update? Note that the next patch is coming out this Tuesday :smile: 

I mean, I don't expect we have schema changes until then, so it's fine. But you get the idea ;)

Reply:
Hi @npepinpe , hmm, I was under the impression that 8.6.10 was our patch release for the month and that 8.6.11 will come only next month. I guess it's up to how[ this task](https://github.com/camunda/camunda-docs/issues/5118) is prioritized by @RomanJRW

Reply:
No, we're creating a patch today. However, there are no schema changes, so it's not urgent for today.

That said, we can always release out of band patches, which is why doing it sooner is better. But again, another caveat - the customer is not in production, so updates are possibly not ultra urgent.